.github/workflows/upgrade.yaml

```diff
@@ -68,15 +68,12 @@ jobs:
       - name: Upgrade Helm version
         run: |
           make docker-buildx \
-            IMG=gatekeeper-e2e:latest \
-            GATEKEEPER_NAMESPACE=${{ matrix.GATEKEEPER_NAMESPACE }}
+            IMG=gatekeeper-e2e:latest
 
           make docker-buildx-crds \
-            CRD_IMG=gatekeeper-crds:latest \
-            GATEKEEPER_NAMESPACE=${{ matrix.GATEKEEPER_NAMESPACE }}
```

Comment:
should we be getting rid of this field?

Reply:
`matrix.GATEKEEPER_NAMESPCE` does not exists in the workflow. Nor the variable `GATEKEEPER_NAMESPACE` is being used in `make docker-buildx-crds` rule.

Reply:
got it, thanks